packages/autorest.typescript/src/restLevelClient/generateClientDefinition.ts

```diff
@@ -1,386 +1,48 @@
-import {
-  CodeModel,
-  Operation,
-  ParameterLocation,
-  ImplementationLocation,
-  Response
-} from "@autorest/codemodel";
-import { isEqual } from "lodash";
+import { CodeModel } from "@autorest/codemodel";
 
-import {
-  gerOperationSuccessStatus,
-  getResponseTypeName
-} from "./operationHelpers";
-
-import {
-  CallSignatureDeclarationStructure,
-  Project,
-  SourceFile,
-  StructureKind,
-  Writers
-} from "ts-morph";
+import { Project } from "ts-morph";
 import * as path from "path";
 
 import { getAutorestOptions } from "../autorestSession";
-import { CasingConvention, NameType, normalizeName } from "../utils/nameUtils";
+import { NameType, normalizeName } from "../utils/nameUtils";
 import { getLanguageMetadata } from "../utils/languageHelpers";
-import {
-  buildMethodDefinitions,
-  getOperationParameters,
-  getPathParamDefinitions
-} from "./helpers/operationHelpers";
-import {
-  generateMethodShortcuts,
-  REST_CLIENT_RESERVED
-} from "./generateMethodShortcuts";
-import { Methods, PathParameter, Paths, ResponseTypes } from "./interfaces";
-import { isLongRunningOperation } from "./helpers/hasPollingOperations";
+import { transformPaths } from "./transforms/transformPaths";
+import { Paths, buildClientDefinitions } from "@azure-tools/rlc-codegen";
 export let pathDictionary: Paths = {};
```

Comment:
I see pathDirectionary is being used in several places, should we make it part of RLCModel instead of export it from generateClientDefinition.ts file ? it looks a little strange if we need to import something from another generateXXX.ts file.

Reply:
Yeah it is now part of the RLC Model. I'll see if I can remove this global